src/components/Timer/Timer.tsx

```diff
@@ -5,6 +5,8 @@ import {Actions} from "store/action";
 import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
 import "./Timer.scss";
 import {useTranslation} from "react-i18next";
+import {Toast} from "utils/Toast";
+import i18n from "i18n";
```

Comment:
```suggestion
```
We are already using `useTranslation` imported in line 7, hence you can reuse that.